### PR TITLE
Issue 299 fixes

### LINKED ISSUE #299: Update contributing guide
Need to update lab name and fix a bunch of broken links (so, check all the links).

### PR DESCRIPTION
**Pull request by stiber**
closes #299

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 If you're making modifications that you'd like to be merged into our code base, then please see the Workflow section, below. When unsure, contact us ahead of time.
 
-## For the BioComputing Lab
+## For the Intelligent Networks Lab
 For UW Bothell students interested in working in the BCL, we use a [shared repository development model](https://help.github.com/articles/about-collaborative-development-models/). If you're interested in contributing directly to this project, then please contact [Prof. Michael Stiber](mailto:stiber@uw.edu) and read the information below.
 
 ## Workflow
```
Comment (jardiamj) [inline CONTRIBUTING.md:13]: The link for "Managing your work on GitHub" doesn't seem correct: https://help.github.com/categories/managing-your-work-on-github/
It sends me to the github issues doc page (https://docs.github.com/en/issues). I am not sure what this link is supposed to be.

Comment (VivekGandhi54) [reply CONTRIBUTING.md:13]: https://web.archive.org/web/20180708165903/https://help.github.com/categories/managing-your-work-on-github/
This is what it used to be.

Comment (stiber) [reply CONTRIBUTING.md:13]: It seems that this is still a very similar page. In fact, I would say that the current title is not so good. But I'll change the link text...

Comment (jardiamj) [reply CONTRIBUTING.md:13]: This is exactly what I was about to write. It is the exact same content, just a different title.

**Review by jardiamj — APPROVED** [review]

**Pull request merged**